tests/pages/loginPage.js

```diff
@@ -0,0 +1,27 @@
+const { expect } = require('@playwright/test');
+
+exports.LoginPage = class LoginPage {
+
+  /**
+   * @param {import('@playwright/test').Page} page
+   */
+    constructor(page) {
+        this.page = page;
+        this.email = page.locator('[id="user"]');
+        this.password = page.locator('#password');
+        this.enterEm = page.locator('[id="login"]');
```

Comment:
This looks like a button, so you should call it emailButton or something like that.